int64 sent
int64 dropped
int64 late
int64 unassigned_drops